unicorn_mode/samples/speedtest/rust/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "unicornafl_harness"
+name = "unicornafl"
```

Comment:
Why rename the package, here? I prefer the current name or just `harness`

Reply:
Oops, my bad. Sure I agree with you, I wanted to rename it to `harness` like the other ones, but I edited the string wrong. Just fixed.